### PR TITLE
Build csv graph fix

### PR DESCRIPTION
**Pull request by dantavori**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: .gitlab/ci/on-push.yml [hunk 1]
```diff
@@ -160,27 +160,27 @@ validate-content-conf:
       fi
     - section_end "Collect Tests"
 
-    - section_start "Prepare Content Packs for Testing"
+    - section_start "Prepare Content Packs for Testing" --collapsed
```
Comment (ilaner) [inline .gitlab/ci/on-push.yml:163]: @esharf asked for this part to be not collapsed

**Comment by ilaner** [comment]
@esharf 
We forgot to set TEST_UPLOAD=`false` in the schedule, and updated it only in the script.
We think it's better to set it the other way around: to set it to TEST_UPLOAD=`true` only in the test upload, that way we need to maintain this variable only in the test upload script.

File: .gitlab/ci/.gitlab-ci.yml [hunk 2]
```diff
@@ -42,7 +42,7 @@ variables:
   DEMISTO_CONNECTION_POOL_MAXSIZE: "180" # see this issue for more info https://github.com/demisto/etc/issues/36886
   SDK_REF: "master" # The default sdk branch to use
   OVERRIDE_ALL_PACKS: "false"
-  TEST_UPLOAD: "true"
+  TEST_UPLOAD: "false"
```
Comment (esharf) [inline .gitlab/ci/.gitlab-ci.yml:45]: already fixed in the schedule
```suggestion
  TEST_UPLOAD: "true"
```

File: Utils/trigger_test_upload_flow.sh [hunk 3]
```diff
@@ -156,6 +156,7 @@ if [ -n "$_gitlab" ]; then
     --form "variables[STORAGE_BASE_PATH]=${_storage_base_path}" \
     --form "variables[OVERRIDE_ALL_PACKS]=${_override_all_packs}" \
     --form "variables[CREATE_DEPENDENCIES_ZIP]=${_create_dependencies_zip}" \
+    --form "variables[TEST_UPLOAD]=true" \
```
Comment (esharf) [inline Utils/trigger_test_upload_flow.sh:159]:
```suggestion
```

File: Utils/gitlab_triggers/trigger_upload_flow_build.sh [hunk 4]
```diff
@@ -99,6 +99,5 @@ curl -k -v --request POST \
   --form "variables[GCS_MARKET_V2_BUCKET]=${_bucket_v2}" \
   --form "variables[GCS_MARKET_XPANSE_BUCKET]=${_bucket_xpanse}" \
   --form "variables[IFRA_ENV_TYPE]=Bucket-Upload" \
```
Comment (esharf) [inline Utils/gitlab_triggers/trigger_upload_flow_build.sh:101]: remove this change please

**Review by esharf — CHANGES_REQUESTED** [review]

File: .gitlab/ci/on-push.yml (hunk 1 above)
Comment (dantavori) [reply .gitlab/ci/on-push.yml:163]: why not?

Comment (esharf) [reply .gitlab/ci/on-push.yml:163]: we would like to see the errors right when we open the job without start digging.
this should be reserved for parts that we are sure that they are stable and they have very long outputs.

Comment (dantavori) [reply .gitlab/ci/on-push.yml:163]: There are too many log lines in this section, you always need to enter the raw logs to see everything.
Another approach - reduce the number of info logs @ilaner

Comment (esharf) [reply .gitlab/ci/on-push.yml:163]: O

**Pull request closed without merging**